pkg/plugins/trivy/plugin.go

```diff
@@ -304,9 +338,10 @@ func NewTrivyConfigAuditPlugin(clock ext.Clock, idGenerator ext.IDGenerator, obj
 func (p *plugin) Init(ctx trivyoperator.PluginContext) error {
 	return ctx.EnsureConfig(trivyoperator.PluginConfig{
 		Data: map[string]string{
-			keyTrivyImageRef:                  "ghcr.io/aquasecurity/trivy:0.30.0",
-			keyTrivySeverity:                  "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL",
-			keyTrivyMode:                      string(Standalone),
+			keyTrivyImageRef: "ghcr.io/aquasecurity/trivy:0.30.0",
+			keyTrivySeverity: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL",
+			keyTrivyMode:     string(Standalone),
+			keyTrivyAdditionalVulnerabilityReportFields: "",
```

Comment:
we need to remove this key/value of `keyTrivyAdditionalVulnerabilityReportFields: "",`on the `Init` method otherwise it will not write the additional fields when deploying via helm chart :
how to test it : 
```
helm install trivy-operator ./deploy/helm \
  --namespace trivy-system \
  --create-namespace \
  --set="trivy.additionalVulnerabilityReportFields=Description\,Links\,CVSS" \
  --set="trivy.ignoreUnfixed=true"
```

Reply:
Done! )